packages/debridge/src/tokens.ts

```diff
@@ -0,0 +1,31 @@
+import { SwapTokenMetadata } from './debridge.types';
+
+export const CHAINS_MAP = {
+  ethereum: 1,
+  optimism: 10,
+  arbitrum: 42161,
+  polygon: 137,
+  binance: 56,
+  avalanche: 43114,
+} as const;
+
+type Chain = keyof typeof CHAINS_MAP;
+type Tokens<K extends Chain = Chain> = Record<
+  string,
+  Partial<{ [k in K]: SwapTokenMetadata & { chainId: (typeof CHAINS_MAP)[k] } }>
+>;
+
+export const tokens = {
+  WBTC: {
```

Comment:
as discussed, let the chain name be the key and all its supporting tokens inside with token symbol as the key.